model/cluster_utility.go

```diff
@@ -73,6 +75,9 @@ var DefaultUtilityVersions map[string]*HelmUtilityVersion = map[string]*HelmUtil
 	PgbouncerCanonicalName: {Chart: "1.2.0", ValuesPath: ""},
 	// PromtailCanonicalName defines the default version and values path for the Helm chart
 	PromtailCanonicalName: {Chart: "4.2.0", ValuesPath: ""},
+	// RtcdDefaultVersion defines the default version for the Helm chart
+	RtcdCanonicalName: {Chart: "1.0.0", ValuesPath: ""},
+	// KubecostDefaultVersion defines the default version for the Helm chart
```

Comment:
Don't think this is needed based on format of other Canonical Names and the comment already exists for the `KubecostCanonicalName` element.